Memory_management_algorithm/best_fit.cpp

```diff
@@ -0,0 +1,102 @@
+#include <bits/stdc++.h>
```

Comment:
Change the file name to Memory_Management_Algorithm/Best_Fit.cpp 
Make a similar change in the whole files

Reply:
Memory_Management_Algorithm/Best_Fit.cpp